ci/cargo-test-bpf.sh

```diff
@@ -31,7 +31,7 @@ if [[ -r $run_dir/Cargo.toml ]]; then
     # Build/test just one BPF program
     set -x
     cd $run_dir
-    cargo +"$rust_stable" test-sbf -- --nocapture
+    cargo +"$rust_stable" test-sbf --arch bpf -- --nocapture
```

Comment:
You don't need to specify bpf architecture. It's the default. The option was added to test sbfv2 architecture as means to test the changes in the new architecture.